Mage.Tests/src/test/java/org/mage/test/serverside/base/impl/CardTestPlayerAPIImpl.java

```diff
@@ -2148,6 +2154,21 @@ public void addTarget(TestPlayer player, TestPlayer targetPlayer, int timesToCho
         }
     }
 
+    /**
+     * Sets players as target
+     */
+    public void addTarget(TestPlayer player, TestPlayer... targetPlayers) {
+        String target = "targetPlayer=";
+        for (int i = 0; i < targetPlayers.length; ++i) {
+            if (i > 0) {
+                target += "^";
```

Comment:
That method must be removed can cause bugs (see above about dynamic targets). Use addTarget per player.

Reply:
@Susucre are you planning to fix that?

Reply:
Related issue #11666